cmd/sonobuoy/app/worker.go

```diff
@@ -112,51 +112,42 @@ func loadAndValidateConfig() (*plugin.WorkerConfig, error) {
 }
 
 func runGatherSingleNode(cmd *cobra.Command, args []string) {
-	cfg, err := loadAndValidateConfig()
-	if err != nil {
-		errlog.LogError(err)
-		os.Exit(1)
-	}
-
-	client, err := getHTTPClient(cfg)
```

Comment:
Fixed these repetitive sections while I was here. The only thing different between these methods were the URLs so I extracted that logic.